eZ/Publish/Core/Repository/Strategy/ContentThumbnail/Field/ContentFieldStrategy.php

```diff
@@ -12,15 +12,31 @@
 use eZ\Publish\API\Repository\Values\Content\Thumbnail;
 use eZ\Publish\Core\Base\Exceptions\NotFoundException;
 use eZ\Publish\SPI\Repository\Strategy\ContentThumbnail\Field\ThumbnailStrategy;
+use Traversable;
 
 final class ContentFieldStrategy implements ThumbnailStrategy
```

Comment:
You should probably add some test coverage for this class.

Reply:
Done